src/main/java/org/carlmontrobotics/robotcode2023/subsystems/Arm.java

```diff
@@ -15,148 +15,110 @@
-        return getKg() * getCoM().getAngle().getCos() + armFeed.calculate(vel, accel) + armPID.calculate(armEncoder.getPosition(), goalPosRad[ARM]);
+
+    public double driveArm(double vel, double accel, boolean usePID) {
+        if (Math.abs(ARM_ANGLE_MAX_RAD - getArmPos()) < ARM_ANGLE_TOLERANCE_RAD || Math.abs(ARM_ANGLE_MIN_RAD - getArmPos()) < ARM_ANGLE_TOLERANCE_RAD) {
+            double armFeedVolts = getKg() * getCoM().getAngle().getCos() + armFeed.calculate(0, 0);
+            double armPIDVolts = armPID.calculate(armEncoder.getPosition(), goalPosRad[ARM]);
+            // TODO: REMOVE WHEN DONE WITH TESTING (ANY CODE REVIEWERS, PLEASE REJECT MERGES
+            // TO MASTER IF THIS IS STILL HERE)
+            SmartDashboard.putNumber("ArmFeedVolts", armFeedVolts);
+            SmartDashboard.putNumber("ArmPIDVolts", armPIDVolts);
+            return (usePID ? armFeedVolts + armPIDVolts : armFeedVolts);
+        }
+        double armFeedVolts = getKg() * getCoM().getAngle().getCos() + armFeed.calculate(vel, accel);
+        double armPIDVolts = armPID.calculate(armEncoder.getPosition(), goalPosRad[ARM]);
+        SmartDashboard.putNumber("ArmFeedVolts", armFeedVolts);
+        SmartDashboard.putNumber("ArmPIDVolts", armPIDVolts);
+        return (usePID ? armFeedVolts + armPIDVolts : armFeedVolts);
     }
-    public double driveWrist(double vel, double accel) {
-        if (Math.abs(WRIST_ANGLE_MAX - getWristPos()) < WRIST_ANGLE_TOLERANCE)
-            return wristFeed.calculate(getWristPosRelativeToGround(), 0, 0) + wristPID.calculate(getWristPos(), goalPosRad[WRIST]);
-        return wristFeed.calculate(getWristPosRelativeToGround(), vel, accel) + wristPID.calculate(getWristPos(), goalPosRad[WRIST]);
+
+    public double driveWrist(double vel, double accel, boolean usePID) {
+        if (Math.abs(WRIST_ANGLE_MAX_RAD - getWristPos()) < WRIST_ANGLE_TOLERANCE_RAD || Math.abs(WRIST_ANGLE_MIN_RAD - getArmPos()) < ARM_ANGLE_TOLERANCE_RAD) {
+            double wristFeedVolts = wristFeed.calculate(getWristPosRelativeToGround(), 0, 0);
+            double wristPIDVolts = wristPID.calculate(getWristPos(), goalPosRad[WRIST]);
+            // TODO: REMOVE WHEN DONE WITH TESTING (ANY CODE REVIEWERS, PLEASE REJECT MERGES
+            // TO MASTER IF THIS IS STILL HERE)
+            SmartDashboard.putNumber("WristFeedVolts", wristFeedVolts);
+            SmartDashboard.putNumber("WristPIDVolts", wristPIDVolts);
+            return (usePID ? wristFeedVolts + wristPIDVolts : wristFeedVolts);
+        }
+        double wristFeedVolts = wristFeed.calculate(getWristPosRelativeToGround(), vel, accel);
+        double wristPIDVolts = wristPID.calculate(getWristPos(), goalPosRad[WRIST]);
+        SmartDashboard.putNumber("WristFeedVolts", wristFeedVolts);
+        SmartDashboard.putNumber("WristPIDVolts", wristPIDVolts);
+        return (usePID ? wristFeedVolts + wristPIDVolts : wristFeedVolts);
     }
```

Comment:
In each of these 2 methods, the only difference between what is in the if and what isn't is that vel and accel are 0. Please don't duplicate code unnecessarily.